recipes/dj-static/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "dj-static" %}
+{% set version = "0.0.6" %}
+{% set build = 2 %}
```

Comment:
Unless you are trying to avoid a collision with a version present in the default channel let's set this one to `0`.